addons/auth_signup/data/ir_config_parameter_data.xml

```diff
@@ -5,5 +5,7 @@
         <function model="ir.config_parameter" name="set_param" eval="('auth_signup.invitation_scope', 'b2c')"/>
         <!-- activate Reset password by default -->
         <function model="ir.config_parameter" name="set_param" eval="('auth_signup.reset_password', 'True')"/>
+        <function model="ir.config_parameter" name="set_param" eval="('auth_signup.signup.validity.hours', 7*24)"/>
+        <function model="ir.config_parameter" name="set_param" eval="('auth_signup.reset_password.validity.hours', '4')"/>
```

Comment:
I am not sure we should create these by default:
- These defaults are alreadty set we calling `get_param` for these
- Not sure I want to expose by default such a parameter, for which 99.9% of users are not interested in.